README.md

```diff
@@ -41,6 +41,46 @@ exports.config = {
 
 `outputDir` defaults to `./allure-results`. After a test run is complete, you will find that this directory has been populated with an `.xml` file for each spec, plus a number of `.txt` and `.png` files and other attachments.
 
+## Custom attachments
+
+You can save custom attachments to tests/steps by calling `process.send` to `allure:attachment` within your hook:
+
+```js
+afterStep: function (step) {
+    var serverLogs = getServerLogsSomehow();
+    
+    process.send({
```

Comment:
I'd rather not to call `process.send`, directly, because it is an implementation detail that may be changed.

Instead of this, we can create an extra `runtime.js` file that should be imported in the test code

```
export default {
  createAttachment(name, content, type) {
    process.send({
        event: 'allure:attachment',
        name,  content, type
    });
  }
}
```

it will be better for all users to have a stable API, so we can change implementation inside